frappe/email/doctype/notification/notification.py

```diff
@@ -363,18 +392,28 @@ def get_receiver_list(self, doc, context):
 
 			# For sending messages to the owner's mobile phone number
 			if recipient.receiver_by_document_field == "owner":
-				receiver_list += get_user_info([dict(user_name=doc.get("owner"))], "mobile_no")
+				receiver_list += get_user_info([dict(user_name=doc.get("owner"))], user_field)
```

Comment:
Can maybe add a fallback here incase `user_field` is not passed?

(not sure about the usages of this method so not 100% sure if its required, I do see atleast one other call: https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/frappe/.*+get_receiver_list&patternType=standard&sm=0&groupBy=repo)